public/static/locales/en/me.json

```diff
@@ -186,5 +186,13 @@
   "cancelOnSelectedDate": "Select Date of cancellation",
   "conservation-donation": "Conservation Donation",
   "funds-donation": "Donation",
-  "bouquet-donation": "Bouquet Donation"
-}
\ No newline at end of file
+  "bouquet-donation": "Bouquet Donation",
+  "transferDetails": "Transfer Details",
+  "beneficiary": "Beneficiary",
+  "donationNote": {
+    "offline-offline": "Bank transfers usually take 1–5 business days to reach us. As soon as your transfer arrives we will send you a confirmation email with your receipts (and your certificate).",
+    "stripe-sepa_debit": "SEPA payments take 2-14 business days (usually 3-5 days). As soon as it arrives we will send you a confirmation email with your tree certificate and raise the tree counter.",
+    "stripe-sofort": "SOFORT payments take 2-14 business days (usually 2-3 days). As soon as it arrives we will send you a confirmation email with your tree certificate and raise the tree counter.",
```

Comment:
Are these messages only used for tree donation speaking from certificates and tree counters? @sagararyal as I understood we now have to carefully check the type of donation when speaking about certificates or tree counters, right?

Reply:
@norbertschuler correct, this should be adapted for donation types.